com.fugro.roames.displaz.json

```diff
@@ -39,15 +37,17 @@
       ],
       "build-commands": [
         "mkdir build_external",
-        "pushd build_external && cmake ../thirdparty/external && make && popd",
+        "pushd build_external && cmake ../thirdparty/external && make -j ${FLATPAK_BUILDER_N_JOBS} && popd",
         "mkdir build",
-        "pushd build && cmake -DCMAKE_INSTALL_PREFIX=/app .. && make install && popd",
-        "mkdir -p /app/share/icons/hicolor/256x256/apps/",
-        "mkdir -p /app/share/applications/",
-        "mkdir -p /app/share/metainfo/",
-        "cp src/resource/displaz_icon_256.png            /app/share/icons/hicolor/256x256/apps/com.fugro.roames.displaz.png",
-        "cp flatpak/com.fugro.roames.displaz.desktop     /app/share/applications/com.fugro.roames.displaz.desktop",
-        "cp flatpak/com.fugro.roames.displaz.appdata.xml /app/share/metainfo/com.fugro.roames.displaz.appdata.xml"
+        "pushd build && cmake -DCMAKE_INSTALL_PREFIX=/app .. && make -j ${FLATPAK_BUILDER_N_JOBS} install && popd",
+        "install -TDm644 src/resource/icons/hicolor/scalable/apps/displaz.svg ${FLATPAK_DEST}/share/icons/hicolor/scalable/apps/${FLATPAK_ID}.svg",
```

Comment:
Ideally, all of the files should be installed using cmake. They can stay here for now, but they are a requirement for other linux packages too. At least, if you want to give a great linux experience :)

Reply:
Yes, agreed.  I'm not the official maintainer for displaz, but I think having various patches for flatpak and in general, will both be coming up.  I'm already pretty happy with the way this flatpak is coming together.  Working through all these details is helpful, thanks for that.